src/Domain.LinnApps/Requisitions/RequisitionManager.cs

```diff
@@ -618,6 +618,20 @@ public async Task<RequisitionHeader> Validate(
                     throw new CreateRequisitionException($"PO {document1Number} is FIL Cancelled!");
                 }
             }
+            else if (function.PartSource == "C" && function.Document1Required())
+            {
+                if (document1Type != "C" || document1Number == null)
+                {
+                    throw new CreateRequisitionException("Function requires a credit note");
```

Comment:
all the other checks on, e.g. document1Number being present when stores function requires it, are in the RequistionHeader constructor which is probably technically a better (or at least more consistent) place for this one too

Reply:
the document1Number check is a bit of a duplication but the actual check here is that document type is the correct type i.e. C which is a bit harder to generalise into the requisition header but I can take out the document1number duplication